themes/_core/js/common.js

```diff
@@ -28,3 +28,23 @@ export function psShowHide () {
   $('.ps-shown-by-js').show();
   $('.ps-hidden-by-js').hide();
 }
+
+/**
+ * Get request parameters
+ * @param preview
+ * @returns {*}
+ */
```

Comment:
Please write a meaningful JSDoc.

As a general rule, I consider that if a developer needs to read the implementation in order to understand what a function does and how to use it, it means that the documentation is incomplete.

Here, you are missing:
- **A meaningful description:** a description which is exactly the same as the function name doesn't add any information.
- The parameter's data type.
- A description for the parameter (what's it for? is it optional? does it have a default value?).
- A more precise type for the return value (I think it's `string|null` in your case)
- An (optional) description for the return value.

You can find more information about JSDoc here: http://usejsdoc.org/